presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -33,6 +33,11 @@ Map Functions
 
     Returns all the values in the map ``x``.
 
+.. function:: map_union(x<K,V>) -> map<K,V>
```

Comment:
This function is an aggregation, so it should go in the aggregations chapter in the "Map Aggregate Functions" section.